### PR TITLE
remove ansible_version from the API config and metrics endpoints

### PR DESCRIPTION
**Pull request by ryanpetrello**
AWX no longer includes Ansible on the control plane and there is no
"global" version of Ansible aside from what's configured at the
Execution Environment level

see: https://github.com/ansible/awx/issues/9472

### CODE REVIEW

File: awx/main/analytics/collectors.py [hunk 1]
```diff
@@ -52,7 +52,6 @@ def config(since, **kwargs):
         'instance_uuid': settings.SYSTEM_UUID,
         'tower_url_base': settings.TOWER_URL_BASE,
         'tower_version': get_awx_version(),
-        'ansible_version': get_ansible_version(),
```
Comment (ryanpetrello) [inline awx/main/analytics/collectors.py:55]: cc @Ladas

Comment (ryanpetrello) [reply awx/main/analytics/collectors.py:55]: @beeankha even though this isn't reported here anymore, it's now part of the analytics payload we ship for jobs (per-job) 🎉 https://github.com/ansible/awx/pull/9598

**Review by Ladas — APPROVED** [review]
:+1: makes sense, I'll add issue on AA side, so we deal with this change

**Review by Spredzy — APPROVED** [review]
Looks good, (Minus test failing)

**Review by beeankha — APPROVED** [review]
LGTM! :+1:

**Review by jakemcdermott — APPROVED** [review]
The code removal lgtm & I ran this locally to do some light testing in the places we used to reference or indirectly reference this variable. I didn't find anything notable.

Let's kick off an e2e just to be sure then :+1: from me.

cc @unlikelyzero @tiagodread @akus062381

**Review by jakemcdermott — APPROVED** [review]
[branch results](https://dashboard.cypress.io/projects/ui2mq7/runs/5065/overview) match [devel results](https://dashboard.cypress.io/projects/ui2mq7/runs/5062/overview)

**Pull request merged**